base-deploy

```diff
@@ -87,7 +87,7 @@ for token in $tokens; do
         . "load-ilk-$(echo "$token" | tr '[:upper:]' '[:lower:]')-$(echo "$ilk" | tr '[:upper:]' '[:lower:]')-$(seth chain)"
         rm "load-ilk-$(echo "$token" | tr '[:upper:]' '[:lower:]')-$(echo "$ilk" | tr '[:upper:]' '[:lower:]')-$(seth chain)"
 
-        if [ "${token}" != "ETH" ] && [ $newToken = "true" ]; then
+        if [ "$token" != "ETH" ] && [ $newToken == "true" ]; then
```

Comment:
```suggestion
        if [[ "$token" != "ETH" ]] && [] $newToken == "true" ]]; then
```

Reply:
If using `[[` and `]]` why not: `[[ "$token" != ETH && "$newToken" == true ]]`

Reply:
Absolutely, feel free to suggest.

Reply:
Yes, sorry I'm on mobile ftm.

Reply:
I was wondering that. If it was better `[[` or just `[`. From the quick read I had done `[` was more backward compatible and the only thing we needed to pay attention was quoting the variables. But if you guys think `[[` is better, we should change it everywhere. I'll take care of it.

Reply:
Made the changes to use `[[` everywhere. @icetan I also did it in a couple of places in `setup-env.sh` could you please double check I've not broken anything?